webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -37,6 +36,23 @@ export class TargetAggregatesService {
     private ngZone:NgZone,
   ) { }
 
+  private readonly NBR_MONTHS = 3;
+  private readonly INTERVAL_TAG_FORMAT = 'Y-MM';
+
+  private getIntervalTag(targetInterval) {
+    return moment(targetInterval.end).format(this.INTERVAL_TAG_FORMAT);
+  }
+
+  private getCurrentInterval(settings) {
```

Comment:
```suggestion
  private getCurrentInterval(appSettings) {
```
I guess this is app settings and not user settings, the parameter name can be like this for clarification